sqrl-planner/sqrl-planner-local/src/main/java/com/datasqrl/loaders/ModuleLoaderComposite.java

```diff
@@ -0,0 +1,26 @@
+package com.datasqrl.loaders;
+
+import com.datasqrl.canonicalizer.NamePath;
+import com.datasqrl.module.SqrlModule;
+import java.util.List;
+import java.util.Optional;
+import lombok.Builder;
+import lombok.Singular;
+import lombok.Value;
+
+@Value
+@Builder
+public class ModuleLoaderComposite implements ModuleLoader {
```

Comment:
Maybe unity all of these classes so there is only one modular loader and only one chain of responsibility pattern. This is going to be hard to wrangle (which module loader is loaded?)